solidity/ecdsa/yarn.lock

```diff
@@ -630,6 +630,14 @@
     "@openzeppelin/contracts" "^4.3.2"
     "@thesis/solidity-contracts" "github:thesis/solidity-contracts#4985bcf"
 
```

Comment:
Once #2953 is merged, we need to update `@keep-network/random-beacon` dependency to eliminate the sortition pool entry above.

Reply:
Let's merge https://github.com/keep-network/keep-core/pull/2956 first. Then, once a new `@keep-network/random-beacon` NPM package is published, I will update this PR.

Reply:
Done in https://github.com/keep-network/keep-core/pull/2954/commits/bc00d15d64a9c3e8333ddc256d87243412aa8f6e.